README.md

```diff
@@ -64,9 +64,12 @@ A curated list of anime streaming/downloading websites.
 * [station.f-n.me](https://station.f-n.me/)
 * [thehylia.com](https://anime.thehylia.com/)
 * [watchcartoononline.io](https://www.watchcartoononline.io/)
+* [viewster.com](https://www.viewster.com/)
 
 ### Anime (mobile app)
 * [Anime HD Watch - Kissanime](https://play.google.com/store/apps/details?id=com.pg.kissanimenew&hl=en)
+* [#Animania](http://www.mobi24.net/#anime)
+* [Popcorn Time](https://popcorntime.sh/)
```

Comment:
Does it provide animes as well? Please share ss. :smile:

Reply:
Screenshot for animania is available on the mentioned link (http://www.mobi24.net/#anime).

For popcorn time any content available on torrent can be streamed. Thus, if the show is available on torrent, it is available on popcorn time.

Reply:
> Screenshot for animania is available on the mentioned link (http://www.mobi24.net/#anime).
> 
> For popcorn time, any content available on torrent can be streamed. Thus, if the show is available on torrent, it is available on popcorn time.

So popcorn seems like a generic solution and not centered to anime so I'm not sure if this is a right pick for this list, please remove it as well. :smile: